### DISCUSSION TITLE
New Solution AstarionReview: Attestation-Based Dapp Rating System

### DISCUSSION THREAD

**Discussion by ASTARIONCrypto** [ethereum-optimism/ecosystem-contributions#171] [Builds]
Hey from the Astarion Team,

we are working at the solution for the Attestation-Based Dapp Rating System idea as described at
https://contribute.optimism.io/issue/attestation-based-app-rating

Take a look at:
https://github.com/ASTARIONCrypto/AstarionReview

We already implemented many porjects on Starknet, Scroll and Blast and now we want to create our Dapp on Optimism.

We want to implement this points:

Allow anyone to register a project for review.
Allow users to leave reviews on projects.
Allow users to see the reviews that others have left on a project.
Allow users to search through projects.
Allow everyone to see all reviews from a reviewer.
Some kind of "accuracy rating" based on how many other reviewers rated the project the same as you.

Everyone is welcome to help to discuss this new idea with us!

Astarion: 

https://astarion.net/

**Comment by Chakerbijo** [comment]
Cool
